docs/1.3_to_1.4_migration_guide.md

```diff
@@ -23,3 +23,19 @@ Migration scripts, which would be called automatically during pool upgrade proce
 
  - **From levelDb to Rocksdb**: 
     [1_3_396_to_1_3_397.py](https://github.com/hyperledger/indy-node/blob/master/data/migrations/deb/1_3_396_to_1_3_397.py)
+
```

Comment:
Please mention that Pool upgrade should be performed simultaneously for all nodes due to txn format changes.